### DISCUSSION TITLE
Error "/bin/bash: line 1: exec:  #magic___^_^___line: not found" on fresh install

### DISCUSSION THREAD

**Discussion by rishabhgargg** [apache/airflow#37607] [General]
### Official Helm Chart version

1.11.0 (latest released)

### Apache Airflow version

2.7.1

### Kubernetes Version

1.27.6

### Helm Chart configuration

https://gist.github.com/rishabhgargg/e24f16c314a3dcfb436f3bb203ed0f05

### Docker Image customizations

None

### What happened

I did this multiple time on different EKS clusters, all new.

But, for some reason as soon as I apply the chart with the provided `values.yaml` with a change `useStandardNaming: true` (which is the default values.yaml from chart) above, I am getting this error `/bin/bash: line 1: exec:  #magic___^_^___line: not found`.

### What you think should happen instead

Ideally, all pods should be up as soon as the default chart is applied.

### How to reproduce

1. helm repo add apache-airflow https://airflow.apache.org
2. helm upgrade --install airflow apache-airflow/airflow --namespace airflow --create-namespace

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by mbrunnen** [comment]
Comes from yamlfmt: https://github.com/google/yamlfmt/blob/176fffc80d1c77511861c9dfa63fb1471612a35f/internal/hotfix/retain_line_break.go#L28

Setting `retain_line_breaks: false` in `.yamlfmt` worked-around the problem in a totally different project.

See also: https://github.com/google/yamlfmt/issues/86

**Comment by didlawowo** [comment]
meet the same error doesn't know why, first time it was ok then relaunch chart and get this

**Comment by potiuk** [comment]
Sems like a bug in yamlffmt in case of some specific whitespace (End of line) in yaml https://github.com/google/yamlfmt/issues/86 has explanation. Seems you need to reformat your yaml to not have the whitespace that causes it. Nothing specific to Airflow.

Converting it to a discussion.
